tests/data/csv/component-definitions/bp/component-definition.json

```diff
@@ -419,6 +419,20 @@
             "remarks": "rule_set_09"
           }
         ],
+        "protocols": [
+          {
+            "uuid": "0130b4ef-4561-4363-bab5-1e20b9888e49",
+            "name": "https",
+            "title": "Transport Layer Security",
+            "port-ranges": [
+              {
+                "start": 443,
+                "end": 443,
+                "transport": "TCP"
+              }
+            ]
+          }
+        ],
```

Comment:
adding this block to this file caused several test failures. The changes in `scripts/schema_preprocess.py` moved them back to passing.